test/pack/Makefile.am

```diff
@@ -23,7 +23,7 @@ $(top_builddir)/dtpools/libdtpools.la:
 	(cd $(top_builddir)/dtpools && $(MAKE))
 
 testing:
-	for tlist in ${testlists} ; do \
-		$(top_srcdir)/runtests.py --testlist=$${tlist} \
-			--summary=$(top_builddir)/pack/summary.junit.xml ; \
-	done
+	TLIST_COMMA="" ; \
+	for tlist in ${testlists}; do TLIST_COMMA="$${TLIST_COMMA},$${tlist}"; done ; \
```

Comment:
Can't each directory directly append its testlist using a comma instead of such post processing?

Reply:
I am not sure if automake can do that.

Reply:
I see.  This is fine then.  Can you rebase on to the latest `yaksa/master` and run the tests?